lib/image/options/write.ex

```diff
@@ -240,10 +256,10 @@ defmodule Image.Options.Write do
   end
 
   defp image_type_from("", suffix) do
-    {:ok, suffix}
+    {:ok, String.trim_leading(suffix, ".")}
   end
 
-  defp image_type_from(extname, _suffix) do
-    {:ok, extname}
+  defp image_type_from(path, _suffix) do
+    {:ok, path |> String.split(".") |> List.last() |> String.downcase()}
```

Comment:
By downcasing the extension, there is no more error in case the output name is set to `IMG.JPG` for example.